packages/utils/src/utils/asserts.utils.ts

```diff
@@ -1,17 +1,20 @@
 export class InvalidPercentageError extends Error {}
 export class NullishError extends Error {}
 
-export const assertNonNullish: <T>(
+export function assertNonNullish<T>(
```

Comment:
I actually remember explicitly writing this function as a `const` for consistency reasons, given that we do not use `function`. Therefore, I'm not against it remaining that way.